src/components/CavaticaProjectList/CavaticaProjectItem.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Button, Popup, Divider} from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
+
+const CavaticaProjectItem = ({projectNode, unlinkProject, studyId}) => {
+  return (
+    <List.Item>
+      <Icon
+        name={
+          projectNode.projectType === 'DEL'
+            ? 'paper plane outline'
+            : 'sliders horizontal'
+        }
+      />
+      <List.Content>
+        {unlinkProject && studyId ? (
+          <List.Content floated="right">
+            <Popup
+              trigger={
+                <Button
+                  size="mini"
+                  basic
+                  negative
+                  onClick={e => e.stopPropagation()}
+                >
+                  UNLINK PROJECT
+                </Button>
+              }
+              header="Are you sure?"
+              content={
+                <>
+                  The project will remain in Cavatica even you unlink it from
+                  here.
+                  <Divider />
+                  <Button
+                    data-testid="delete-confirm"
+                    negative
+                    fluid
+                    icon={<Icon name="unlink" />}
+                    content="Unlink"
+                    onClick={e => {
+                      e.stopPropagation();
+                      unlinkProject({
+                        variables: {project: projectNode.id, study: studyId},
+                      });
+                    }}
+                  />
+                </>
+              }
+              on="click"
+              position="top right"
+            />
+          </List.Content>
+        ) : (
+          <List.Content floated="right">
+            {projectNode.study ? (
+              <Link to={`/study/${projectNode.study.kfId}/documents`}>
+                {projectNode.study.shortName ||
+                  projectNode.study.name ||
+                  projectNode.study.kfId}
+              </Link>
+            ) : projectNode.deleted ? (
+              <>
+                Deleted <Icon name="warning" inverted bordered color="red" />
+              </>
+            ) : (
+              'Not linked'
+            )}
+          </List.Content>
+        )}
+
+        <List.Header
+          as="a"
```

Comment:
It may be nice to disable the link when using the dropdown selection. It's not very friendly how it opens up a new tab when I click a certain area of the item to select it.